shin-core/src/format/scenario/info.rs

```diff
@@ -39,10 +59,15 @@ impl BustupInfoItem {
     }
 }
 
+/// Represents the metadata for a background music (BGM) track.
 #[derive(Debug, BinRead, BinWrite)]
 pub struct BgmInfoItem {
+    /// The internal name of the BGM track, corresponding to the filename the engine will look for when playing the BGM.
```

Comment:
It's actually a name in the `/bgm` directory, excluding the file extension! Ditto for bustup, picture, mask & stuff below.

Smth like "The internal name of the BGM track, corresponding to the filename the engine will look for in the `/bgm` directory when playing the BGM."?